dev/migrate.ps1

```diff
@@ -6,12 +6,33 @@
 #  in the future and investigate if we can migrate back.
 # docker-compose --profile mssql exec mssql bash /mnt/helpers/run_migrations.sh @args
 
-docker run `
-  -v "$(pwd)/helpers/mssql:/mnt/helpers" `
-  -v "$(pwd)/../util/Migrator:/mnt/migrator/" `
-  -v "$(pwd)/.data/mssql:/mnt/data" `
-  --env-file .env `
-  --network=bitwardenserver_default `
-  --rm `
-  mcr.microsoft.com/mssql-tools `
-  /mnt/helpers/run_migrations.sh @args
+param([switch]$all = $false, [switch]$postgres = $false, [switch]$mysql = $false)
+
+if ($all) {
+  Write-Host "Starting Microsoft SQL Server Migrations"
+}
+
+if ($all -or (!$postgres -and !$mysql)) {
```

Comment:
Mainly because the recommendation in PowerShell is to avoid the default of a switch to be true.

https://docs.microsoft.com/en-us/powershell/utility-modules/psscriptanalyzer/rules/AvoidDefaultValueSwitchParameter?view=ps-modules

What I was going for:

`pwsh ./migrate.ps1` runs as it has in the past, just performing MS SQL Server migrations without outputting "Starting Microsoft SQL Server Migrations"

`pwsh migrate.ps1 -mysql` just runs mysql migration
`pwsh migrate.ps1 -postgres` just runs postgres migrations
`pwsh ./migrate.ps1 -all` runs everything